menghua_co/custom/work_order.py

```diff
@@ -5,10 +5,10 @@
 
 class WorkOrder(Document):
     def after_insert(self):
-        make_manufacturing_order_auto(self.name)
+        make_manufacturing_order_auto(self)
 
 @frappe.whitelist()
-def make_manufacturing_order_auto(doc, method):
+def make_manufacturing_order_auto(doc):
     make_manufacturing_order_auto = frappe.get_cached_value(
```

Comment:
```suggestion
    auto_mo = frappe.get_cached_value(
```
Note: do not use same var name as function, it will be hard to search later. Variable (not field name) no need to be lengthy, it consume space.